drivers/scheduler/k8s/k8s.go

```diff
@@ -776,6 +811,32 @@ func (k *K8s) substituteNamespaceInPVC(pvc *v1.PersistentVolumeClaim, ns string)
 	}
 }
 
+//Set or disable the placement_strategy in the storageclass
+func (k *K8s) substituteVpsNameInStorageClass(sc *storageapi.StorageClass, vps_map map[string]string) {
+	//	sc.Name = namespaceRegex.ReplaceAllString(sc.Name, ns)
+	for k, v := range sc.Parameters {
+		logrus.Infof("[%v] storaegeclass parameters: %v", k, v)
+		newstr := placement1Regex.ReplaceAllString(v, vps_map["placement-1"])
```

Comment:
@naresh we already have support to templating in the torpedo, can we try to use that here? if you have any questions I can help you. check here for an example https://github.com/portworx/torpedo/pull/394

Reply:
@thiguetta  can you review it again

Reply:
made changes around this code area, can you please review it again